lib/dfe/reference_data/degrees.rb

```diff
@@ -7145,6 +7145,12 @@ module Degrees
             synonyms: [],
             dttp_id: '4d8370f0-5dce-e911-a985-000d3ab79618',
             hesa_itt_code: '100750' },
+          'ba1ffe89-73fb-4de8-a48d-12e644ee5687' =>
+          { name: 'Physical Education',
+            synonyms: ['PE'],
+            dttp_id: nil,
+            hesa_itt_code: '100433',
```

Comment:
I'm not sure we should map this...

Reply:
It seems correct _for now_, at least according to the comment on the HECoS guidance? But equally could be nil, which is what would currently happen (although then do ITT providers have to manually look up the code when reporting it to HESA?)